packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -1,24 +1,35 @@
 <template>
-  <div class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center">
-    <span
-      class="flex w-16px items-center justify-center"
-      :data-cy="`git-status-${props.gql.statusType}`"
+  <div
+    class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center"
+    data-cy="git-info-row"
+  >
+    <Tooltip
+      placement="top"
+      class="h-full grid items-center"
+      :disabled="!classes.showTooltip"
     >
-      <span
-        v-if="props.gql.statusType !== 'unmodified'"
-        class="rounded-full border-1 min-w-6px min-h-6px max-w-6px max-h-6px"
-        :class="classes?.indicator"
+      <component
+        :is="classes.icon"
+        :class="classes.iconClasses"
       />
-      <IconGithub
-        v-else
-        class="min-w-16px min-h-16px max-w-16px max-h-16px icon-dark-gray-300 group-hocus:icon-dark-indigo-500"
-      />
-    </span>
+      <template
+        v-if="classes.showTooltip"
+        #popper
+      >
+        <div>
+          <p class="max-w-sm text-sm truncate overflow-hidden">
+            {{ props.gql.subject }}
+          </p>
+          <p class="text-xs">
+            {{ gitTooltipSubtext }}
+          </p>
+        </div>
+      </template>
+    </Tooltip>
     <div
       class="overflow-hidden truncate"
```

Comment:
Looks like we're missing the old gitText classes that were previously applied to the face-up text: 

```js
'text-gray-600 group-hocus:text-indigo-500'
```

I don't think we want the hocus styles anymore based on the current designs, but text-gray-700 (the value now shown in Figma) should be applied here.

Reply:
Fixed